package.json

```diff
@@ -55,28 +53,26 @@
 		"@hashicorp/platform-types": "^0.4.0",
 		"@octokit/rest": "^19.0.5",
 		"@playwright/test": "^1.28.0",
-		"@testing-library/jest-dom": "^5.16.5",
-		"@testing-library/react": "^13.3.0",
+		"@testing-library/jest-dom": "^6.4.2",
+		"@testing-library/react": "^14.2.1",
 		"@testing-library/user-event": "^14.4.3",
 		"@types/cors": "^2.8.12",
 		"@types/google-spreadsheet": "^3.2.2",
-		"@types/jest": "^27.4.1",
 		"@types/jsonwebtoken": "^8.5.9",
 		"@types/node": "^18.18.9",
 		"@types/react": "^18.2.6",
 		"@types/react-dom": "^18.2.4",
 		"@types/semver": "^7.3.9",
 		"@types/shortid": "^0.0.29",
+		"@vitejs/plugin-react": "^4.2.1",
 		"axe-core": "^4.4.1",
-		"babel-jest": "^27.4.6",
 		"cross-env": "^7.0.3",
 		"flat": "^5.0.2",
 		"identity-obj-proxy": "^3.0.0",
-		"jest": "^27.5.1",
-		"jest-raw-loader": "^1.0.1",
+		"jsdom": "^24.0.0",
 		"next-remote-watch": "^2.0.0",
 		"nextjs-bundle-analysis": "^0.5.0-canary.1",
-		"nock": "^13.2.4",
+		"nock": "^14.0.0-beta.5",
```

Comment:
A beta version of `nock` is needed to intercept native `fetch` requests. Previously in Jest I believe it was using `node-fetch`, which the production version of nock is able to intercept.